XRTK.Oculus/Packages/com.xrtk.oculus/Editor/ValidateOculusControllerMappings.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) XRTK. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using UnityEditor;
+using UnityEngine;
+using XRTK.Definitions.InputSystem;
+using XRTK.Definitions.Utilities;
+using XRTK.Oculus.Controllers;
+using XRTK.Utilities;
+
+namespace XRTK.Oculus
+{
+    [InitializeOnLoad]
```

Comment:
Is there a way to make this more generic so we don't have to put something like this in every package?